lib/src/components/FieldGroup/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Label } from '@/components/Label'
+import { classNames, forwardRefWithAs } from '@/utils'
+
+import fieldGroupStyles from './field-group.module.scss'
+import type { FieldGroupProps } from './types'
+
+const cx = classNames(fieldGroupStyles)
+
+export const FieldGroup = forwardRefWithAs<FieldGroupProps, 'fieldset'>(
+  ({ children, className, label, required, ...rest }, ref) => (
+    <fieldset className={cx('root', className)} ref={ref} {...rest}>
+      {label ? (
```

Comment:
`label`(legend) is required for a11y, we can still provide a `hideLabel` prop (similar to the Field component)

Reply:
Yes thanks for spotting: [36a701a](https://github.com/WTTJ/welcome-ui/pull/2925/commits/36a701ab1dc0ada69bc3b30156525ad493e306bd)